Python/Product/Django/TemplateParsing/DjangoFormatDefinitions.cs

```diff
@@ -24,8 +24,8 @@
 namespace Microsoft.PythonTools.Django.TemplateParsing {
     [Export(typeof(EditorFormatDefinition))]
     [ClassificationType(ClassificationTypeNames = DjangoPredefinedClassificationTypeNames.TemplateTag)]
-    [Name(DjangoPredefinedClassificationTypeNames.TemplateTag)]
-    [DisplayName(DjangoPredefinedClassificationTypeNames.TemplateTag)]
+    [Name(DjangoPredefinedClassificationTypeNames.TemplateTag)] // TODO: Localization - this string appears in fonts page in tools options
+    [DisplayName(DjangoPredefinedClassificationTypeNames.TemplateTag)] // TODO: Localization - not sure if this is used, does not affect fonts page
```

Comment:
You shouldn't need to set up a machine - just add a language pack through the installer and change to it in Tools, Options, International Settings.